<launch>
	<!--  xmee_mav Drone type and Commununication Mode -->
	<!-- node pkg="xbee_ros_node" type="xbee_mav" name="xbee_mav" args="master solo" output="screen" -->
	<node pkg="xbee_ros_node" type="xbee_mav" name="xbee_mav" args="master swarm" output="screen" />
	<!-- node pkg="xbee_ros_node" type="xbee_mav" name="xbee_mav" args="slave solo" output="screen" -->
	<!--node pkg="xbee_ros_node" type="xbee_mav" name="xbee_mav" args="slave swarm" output="screen" -->
	<!--  xmee_mav Topics and Services Names -->
	<param name="Xbee_In_From_Buzz" type="str" value="outMavlink" />
	<param name="Xbee_Out_To_Buzz" type="str" value="inMavlink" />
	<param name="Xbee_In_From_Controller" type="str" value="xbee_cmd" />
	<param name="Xbee_Out_To_Controller" type="str" value="mav_dji_cmd" />
  <param name="USB_port" type="str" value="/dev/ttyUSB0" />
  <param name="Baud_rate" type="double" value="115200" />
</launch>
